src/hooks/useAuth.ts

```diff
@@ -11,6 +12,7 @@ const useAuth = () => {
   const [user, setUser] = useState<User | null>(null)
   const [loading, setLoading] = useState(true)
   const [error, setError] = useState<UseUserError | null>(null)
+  const [token, setToken] = useState<string | null>(null);
```

Comment:
I initially used useState for 'token' because I needed it(as you can see in the code from NavigationHeader above) to check if a user is authenticated in my code. However, when I attempted to include 'token' in the return value of the useAuth hook, I faced an error. To resolve this issue, I used useState to store 'token' in the component's state, so it could be accessed and returned. But now upon your feedback, I opted out for only using 'user' to check auth, so  I subsequently removed token from state as I no longer need it for this purpose.